### ISSUE TITLE
Create a tip for using lighthouse

### ISSUE THREAD

**Issue by NothingEverHappens** [fejes713/accessibility-guide#6]
There's a tip suggesting not relying on the lighthouse too much, but many people don't actually know what lighthouse it and how useful it could be. Should we add a `lighthouse` tip and introduce it first?

**Comment by fejes713** [comment]
Yep definitely! I will try to add it over the next day or two. 👍
